lib/page/term_table.rb

```diff
@@ -55,6 +77,8 @@ def group_data
       @group_data
     end
 
+    # A list of people that held a memberships at some point during this term
```

Comment:
Oops, thanks! Fixed in https://github.com/everypolitician/viewer-sinatra/pull/15613/commits/258b63a30245032eeb40471287a1753b8efc434b